docker/cassandra/entry-point

```diff
@@ -12,4 +12,12 @@ do
 done
 )
 
+# Copy over cqlshrc to a default location if defined as ENV
```

Comment:
I think we can make this nicer. It IMO shouldn't be a requirement that users have to define an environment variable and add a file to a config map to get cqlsh to function.

Instead I suggest that we write our own wrapper for `cqlsh` that detects if SSL is enabled, and fetches the CA cert (or just disables trust checking). This will also be necessary once we support authn/authz, since we need to implement shared-secret support.

As a side question, should `cqlsh` be installed the C* container at all? If people want to connect to the cluster, should they instead start a new pod/container that just contains cqlsh?

Reply:
@zegelin we currently use cqlsh now in `cql-readiness-probe`, and this is easily replaceable with either python script or, say, a statically compiled gocql simple program. So we don't have an inherent requirement to have it in the container at all.

Also, if people want to use cqlsh to connect to the cluster, they can either spin a new container with all the certificates and custom configs as they need to, or have it installed locally if they have the network access to the cluster. So, if you think that's a good path to follow, I can replace the readiness script with something else pretty quickly and stop the dependency on `cqlsh`.

Reply:
I did rush it a bit. Clearly, we need any "script" to know about SSL configuration to be able to connect to the node. So having a wrapper around cqlsh might be a useful and simpler solution.

Reply:
In order to get the PR closed, I think we should create a new issue/PR for the wrapper script and solve that one separately.

Reply:
yea, I'm going to remove this part and deal with that in #241